From: Daniel Leidert <daniel.leidert@wgdd.de>
Date: Sat, 22 Feb 2020 01:30:08 +0100
Subject: Remove git from gemspec

---
 rspec-collection_matchers.gemspec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rspec-collection_matchers.gemspec b/rspec-collection_matchers.gemspec
index 0eb56f5..48f3942 100644
--- a/rspec-collection_matchers.gemspec
+++ b/rspec-collection_matchers.gemspec
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
     'source_code_uri'   => 'https://github.com/rspec/rspec-collection_matchers',
   }
 
-  spec.files         = `git ls-files`.split($/)
+  spec.files         = Dir.glob("**/*").select {|v| v !~ /^debian/}
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(spec|features)/})
   spec.require_paths = ["lib"]
